nativelib/src/main/scala/scala/scalanative/runtime/ClibFunctions.scala

```diff
@@ -0,0 +1,15 @@
+package scala.scalanative
+
+import native._
+
+/**
+ * C stdlib functions needed by nativelib
+ *
+ * Note: clib depends on nativelib.
+ */
+
+@extern
+private [scalanative] object clib_string {
```

Comment:
In fact there is a better solution: we should use [LLVM's memset](http://llvm.org/docs/LangRef.html#llvm-memset-intrinsics) here instead.